js/fullWidthTreeView.js

```diff
@@ -39,6 +39,16 @@
     });
   }
 
+  function commandNodeAndChildren($fwTreeView, id, command) {
+    var parent = $fwTreeView.jstree().get_node(id);
+
+    $fwTreeView.jstree(command, id);
+
+    for (let i = 0; i < parent.children.length; i++) {
+      $fwTreeView.jstree(command, parent.children[i]);
+    }
```

Comment:
Do you need to access the children with the index? Would one of these examples without index work the same?

```js
parent.children.forEach((child) => $fwTreeView.jstree(command, child));
```

```js
for (child of parent.children) {
  $fwTreeView.jstree(command, child);
}
```

Reply:
Ah, good call! I'll implement that.